Formula/pdnsrec.rb

```diff
@@ -12,40 +12,37 @@ class Pdnsrec < Formula
     sha256 "a9faf9edf0e71de5e9ec9fa70d27811f353a8451b41746208fbdc0d592aa5910" => :mountain_lion
   end
 
-  depends_on :macos => :lion
+  depends_on "pkg-config" => :build
   depends_on "boost"
-  depends_on "lua" => :optional
+  depends_on "openssl"
+  depends_on "lua"
+  depends_on "gcc" => :linked if MacOS.version <= :mavericks
```

Comment:
> Fails_with is supposed to provide flexibility not lock in a specific compiler dependency.

Sure, ideally. It doesn't though, really, due to the way we handle this for bottles. It won't be any issue once we stop shipping Mavericks bottles in the near future anyway, so meh.